mirrord/agent/env/src/envs.rs

```diff
@@ -51,3 +51,10 @@ pub const POD_IPS: CheckedEnv<Vec<IpAddr>> = CheckedEnv::new("MIRRORD_AGENT_POD_
 ///
 /// Should follow `tracing` format, e.g `mirrord=trace`.
 pub const LOG_LEVEL: CheckedEnv<String> = CheckedEnv::new("RUST_LOG");
+
+/// Container id to get traffic from.
+///
+/// For `Mode::Ephemeral`, this is also used to get the target's `pid`, when this is not
```

Comment:
This makes it sound like it has any other purpose other than ephemeral, which could be true but the env is also called ephemeral so need to have clearer docs.

Reply:
Good one, the naming was very confusing. Changed it and made the docs more explicit.